judgments/views/detail.py

```diff
@@ -70,6 +70,8 @@ def detail(request, judgment_uri):
     context["judgment"] = judgment.content_as_html("")  # "" is most recent version
     context["page_title"] = judgment.name
```

Comment:
Not necessarily for now, but i'm wondering if we could remove this `page_title` and just use `judgment_title` everywhere it's used

Reply:
(we might not actually be able to if eg it's used in the shared page layout)